tutorials/quickstarts/evals_quickstart.ipynb

```diff
@@ -0,0 +1,513 @@
+{
```

Comment:
<p>Can you clear the notebook output? It's pretty noisy and will take up a lot of space in git</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>